### PR TITLE
AWS cloudfront post

### PR DESCRIPTION
**Pull request by njrifkind**
The markdown file for the AWS cloudfront post, to be published Friday
9/25 or Monday 9/28.  Images sent separately via email and have to be
uploaded.

### CODE REVIEW

File: _posts/2015-09-15-aws-cloudfront.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+category :
+tagline: ""
+tags :
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md]: Should we add some tags?

File: _posts/2015-09-15-aws-cloudfront.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+![Content delivery network diagram]({{ BASE_PATH }}/assets/images/aws-cloudfront/aws-cloudfront-diagram.png)
+
+_(Image credit to [How CloudFront Delivers Content](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/HowCloudFrontWorks.html).)_
+
+### Leveraging browser caching
+
+By allowing you to define the period of time for which you want a resource - whether a single image file or an entire webpage - to be considered "current", CloudFront allows you to more fully leverage browser caching to quickly and seamlessly deliver your webpage to visitors. 
+
+It does so by, first, routing the user to the nearest edge location and, second, checking that edge location's cache for a current version of the requested files. If the version available is still consisdered current, it's instantly served up to the visitor; if the cached version is expired, CloudFront sends a request to your origin server for the newest version, serves that file up, and then replaces the expired version available at the edge location.
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md]: "consisdered" => "considered"

File: _posts/2015-09-15-aws-cloudfront.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+How quickly your website loads depends on a host of factors. Some of those factors occur at the level of your website - everything from whether or not your site is [mobile-device friendly](http://googlewebmastercentral.blogspot.com/2015/04/faqs-april-21st-mobile-friendly.html) to the titles of image files included in your blog posts to [how well organized are your CSS files](http://blog.greenfieldhq.com/2014/08/29/front-back/). These and others relating to your [hosting environment](http://blog.greenfieldhq.com/2015/08/21/cloud-hosting/) all are loosely grouped together under the umbrella of _technical SEO_. And this is where a modern and optimized content delivery network, or CDN, can be a huge boon. 
+
+## Technical SEO and CloudFront CDN
+
+As pageload speed is better understood, we've been able to break it down into its individual components. More and more SEO practitioners now are espousing the importance of being able to serve up your website quickly and seamlessly. Here are two of ranking factors that you can improve by optimizing your hosting solution.
+
+### Time To First Byte
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md]: "Time to First Byte" title casing a little off

File: _posts/2015-09-15-aws-cloudfront.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+_[How Website Speed Actually Impacts Search Ranking](https://moz.com/blog/how-website-speed-actually-impacts-search-ranking)_
+
+CloudFront - and other content delivery networks - utilize a global distributed network of servers, referred to as [edge locatons](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html). By allowing visitors to access your site via the edge location nearest to them, response time is lowered and the overall user experience is improved.
+
+![Content delivery network diagram]({{ BASE_PATH }}/assets/images/aws-cloudfront/aws-cloudfront-diagram.png)
+
+_(Image credit to [How CloudFront Delivers Content](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/HowCloudFrontWorks.html).)_
+
+### Leveraging browser caching
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md]: Match title-case with other headers?

File: _posts/2015-09-15-aws-cloudfront.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+By allowing you to define the period of time for which you want a resource - whether a single image file or an entire webpage - to be considered "current", CloudFront allows you to more fully leverage browser caching to quickly and seamlessly deliver your webpage to visitors.
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md]: What does it mean to cache an entire webpage?

File: _posts/2015-09-15-aws-cloudfront.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+Various tactics - everything from [keyword stuffing](https://support.google.com/webmasters/answer/66358?hl=en) to [hiding text](http://searchenginewatch.com/sew/news/2291159/keyword-stuffing-hidden-text-manual-action-google-on-how-to-fix-it#) - at times [have been effective](https://en.wikipedia.org/wiki/Search_engine_optimization#History). One trait these all share, however, is that they were in some way manipulative, and eventually search engines got wise to the actions webmasters were taking to improve their ranking. That's one reason why the first heading on Google's "[Steps to a Google-friendly site](https://support.google.com/webmasters/answer/40349?hl=en)" explicitly encourages us to "Give visitors the information they're looking for."
+
+## Is SEO only about content?
```
Comment (HeroicEric) [inline _posts/2015-09-15-aws-cloudfront.md:28]: Title-case?

File: _posts/2015-09-15-aws-cloudfront.md (hunk 1 above)
Comment (njrifkind) [inline _posts/2015-09-15-aws-cloudfront.md]: Tags - sure. I'd suggest "AWS"; we can add it to the previous post
http://blog.greenfieldhq.com/2015/08/21/cloud-hosting/, too.

File: _posts/2015-09-15-aws-cloudfront.md (hunk 5 above)
Comment (njrifkind) [inline _posts/2015-09-15-aws-cloudfront.md]: edited.

File: _posts/2015-09-15-aws-cloudfront.md (hunk 4 above)
Comment (njrifkind) [inline _posts/2015-09-15-aws-cloudfront.md]: this and "leverage browser caching" were meant to be H3s; changed them to
H2s b/c why not, but not sure if that's what you were referring to.

File: _posts/2015-09-15-aws-cloudfront.md (hunk 2 above)
Comment (njrifkind) [inline _posts/2015-09-15-aws-cloudfront.md]: this and other typos fixed.

**Comment by njrifkind** [comment]
I'm a noob with github - did my changes come through?

**Comment by mikepmunroe** [comment]
Yes, looks like your updates are in the latest version

**Comment by mikepmunroe** [comment]
Overall, I think it's a great post. Looks like Eric caught some grammatical things that you adjusted. We didn't go over images. If you want to email them to me, I can take care of that.

**Comment by njrifkind** [comment]
Great. I'm on my way to the vet to pick up my dog, but will forward the
images as soon as I get home. (Maybe check the email I sent you last night?
Pretty sure I attached them but, admittedly, was pretty frazzled from
dealing with a sick pooch all day.)

I'm glad Eric caught those typos; I need to proofread more closely,
especially if I'm writing in sublime. Won't be an issue again.

**Pull request merged**